Rooms Hero Sample/ClientApp/src/Utils/Utils.ts

```diff
@@ -55,15 +56,14 @@ export const utils = {
         }
         throw new Error('Invalid token response');
     },
-    createRoom: async (identity: string): Promise<string> => {
+    createRoom: async (identity: string, pstnDialOutEnabled: boolean): Promise<string> => {
         const requestOptions = {
             method: 'POST',
             headers: { 'Content-Type': 'application/json' },
             body: JSON.stringify({
```

Comment:
We can simplify this line if the body is empty (was the empty body intentional?)

Reply:
updated